JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -582,7 +582,7 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   return()
 }
 
-.smEstimatesTables         <- function(jaspResults, dataset, options) {
+.smMakeEstimatesTables     <- function(jaspResults, dataset, options) {
   
   models   <- jaspResults[["models"]]$object
   
```

Comment:
I can't comment at the right line 676. I think that lowerRange and upperRange are too generic. You might want to change that to lowerPRange and upperPRange

pFrequency$addColumnInfo(name = "lowerRange", title = gettext("Lower"),     type = "number", overtitle = overtitle)

Reply:
changed